tune_sklearn/tune_search.py

```diff
@@ -258,6 +258,9 @@ class TuneSearchCV(TuneBaseSearchCV):
             determined by 'Pipeline.warm_start' or 'Pipeline.partial_fit'
             capabilities, which are by default not supported by standard
             SKlearn. Defaults to True.
+        time_budget_s (int|float|datetime.timedelta): Global time budget in
+            seconds after which all trials are stopped. Can also be a
+            ``datetime.timedelta`` object.
```

Comment:
We may want to expand this docstring to explain when exactly they are stopped - at the end of the trial, at the start, in the middle, etc.